docs/src/tut/ticket-sales/index.md

```diff
@@ -0,0 +1,274 @@
+- Line 13 declares a `buyTicket` function that our Buyer(s) will be able to call.
+
+Now we've defined our users and the functions they will be allowed, we call `{!rsh} init()` to start stepping through the states of our program.
+
+The first step here is for the Admin to provide the parameters we've declared. This happens in an Local Step. 
+
+[Refresher on Modes of a Reach DApp](https://github.com/reach-sh/reach-lang/discussions/1171)
+```
+load: /examples/ticket-sales/index.rsh
+md5: c425745032273893d106fe3de005f15e
+range: 17-21
+```
+- Line 17 starts the Local Step.
+- Line 18 declassifies the parameters and unpacks them into respective constants.
+- Line 20 publishes those values to the blockchain.
+- Line 21 `{!rsh} commit`s to move us out of consensus step.
+
+Now, the next step is to have the Admin actually pay the non-network tokens into the contract. This needs to happen in a consensus step.
+
+Why then did we move *out* of consensus on Line 21?
+
+The answer is that we need to `{!rsh} publish` the non-network token `tok` before we can `{!rsh} pay` it into the contract. 
+
+Reach supports network tokens by default, but a non-network token, like the one in our program, needs to be taught to Reach before you can pay it in.
+
+The general flow for paying non-network tokens:
+1. Get token info in Local Step.
+2. `{!rsh} publish` token info in Consensus Step.
+3. `{!rsh} commit()`.
+4. `{!rsh} pay` tokens in Consensus Step.
+
+That means our next step is to `{!rsh} pay` the tokens into the contract.
+```
+load: /examples/ticket-sales/index.rsh
+md5: c425745032273893d106fe3de005f15e
+range: 22-23
+```
+- Line 22 pays the `supply` of `tok` from `A` into the contract account. This must be a syntactic tuple where the general structure is `{!rsh} A.pay([networkTokenAmount, [amount, non-networkTokens]]);`.
+- Line 23 notifies the frontend that our contract is ready to start accepting API calls, this can only be called after the first `{!rsh} publish`.
+
+This is a good place to pause in our Reach `rsh` file and get into the frontend `mjs` testing file.
+
+Let's start with some basics.
+```
+load: /examples/ticket-sales/index.mjs
+md5: d2b60e84f70854bc18c26764bdcaf1aa
+range: 1-9
+```
+- Line 1-2 are necessary imports.
+- Line 3 sets a constant for the standard library and hides warning messages.
+- Line 4 sets a constant for the `MAX` number of tickets. Changing this number does so across the entire program.
+- Line 7 launches a new token with a `MAX` supply. Alternatively, you could replace `tickets.id` with a known testnet token id string.
+- Line 9 welcomes you to the ticket sales revolution. Just make sure you [print enough pamphlets.](https://youtu.be/Kb1ztV93dsE)
+
+:::note
+`REACH_NO_WARN: 'Y'`
+
+It is *very* important for you as a programmer to understand the warning messages Reach is giving you before you silence them. Many times, these warnings are indicative of critical DApp design flaws.
```

Comment:
I feel like this is begging you to explain what these ones mean and why you're ignoring them?